projects/plugins/jetpack/extensions/blocks/sharing-buttons/view.js

```diff
@@ -1 +1,52 @@
 import './style.scss';
+
+const services = [
```

Comment:
I'm wondering if getting that info from the server would make future edits easier.

I'm thinking printing the list in a template like:
```
<script>
window.WHATEVER_NAMESPACE.services = '<?php json_encode( $services) ?>';
</script>
```

Then retrieving the list in JS.